cmd/bnsd/x/statefix/model.go

```diff
@@ -0,0 +1,25 @@
+package statefix
+
+import (
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+	"github.com/iov-one/weave/orm"
+)
+
+func init() {
+	migration.MustRegister(1, &ExecutedFix{}, migration.NoModification)
+}
+
+func (ef *ExecutedFix) Validate() error {
+	var errs error
+	errs = errors.AppendField(errs, "Metadata", ef.Metadata.Validate())
+	if len(ef.FixID) == 0 {
```

Comment:
Check the comment above 😸

Reply:
I don't understand. What do you mean?

Reply:
it is related to https://github.com/iov-one/weave/pull/1028#discussion_r337880129. let's move the discussion there